ansible/hadoop-ha.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- hosts: all:!{{ azure_proxy_host }}
```

Comment:
Do you know if it would be possible to only use `start-dfs.sh`?

Reply:
Whilst configuring HA, there is a mixture of tasks like starting services, format Namenode & Zookeeper, bootstrap Standby Namenode - all these tasks follow an order of sequence. This is not possible using `start-dfs.sh` as the script only starts the services. I'm sharing this [doc](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-hdfs/HDFSHighAvailabilityWithQJM.html#Deployment_details) from Apache for more info. Let me know for any further questions? Thanks

Reply:
I was hoping it could be used because the would result in less code in Muchos to maintain.

Reply:
Would it be possible to replace all of the task that start processes in this file with start-dfs.sh, but not the setup/init task? If so would that be a worthwhile simplification?

Reply:
Maybe would still need start-journal task.  Could the following be done and is it worthwhile?

 * start-journal.yml
 * format-nn.yml
 * format-zk.yml
 * start-hdfs.yml
 * start-yarn.yml

Reply:
It's a bit tricky because start-hdfs.yml starts both the namenodes but we need to bootstrap the standby namenode before we start it in-order to successfully configure HA. This is covered in start-nn2.yml. Actually the start-hdfs.yml is not required in-case of hadoop-ha. Happy to take any further comments. Thanks

Reply:
Exactly! Once you configure HA then you can use `start-dfs.sh` / `stop-dfs.sh` scripts to administer the cluster.  Thanks